### ISSUE TITLE
Working with swift 2.3?

### ISSUE THREAD

**Issue by brendanjerwin** [CocoaLumberjack/CocoaLumberjack#762]
I'm trying to get some early testing of things with xcode8 and swift 2.3. Is there a current plan/strategy for that yet?

Thanks for your time.

**Comment by reidmain** [comment]
👍 to this as well. I am looking at updating our application to Swift 2.3 and am encountering serious issues with Carthage being unable to compile CocoaLumberjack using Xcode 8.

**Comment by soleares** [comment]
It compiles with Cocoapods in Xcode8 using a post install hook in the Podfile which adds the SWIFT_VERSION build setting. I'm not sure if Carthage has support for something like this or not. 
```
# Workaround for Swift 2.3
post_install do |installer|
  installer.pods_project.targets.each do |target|
    if target.name == 'CocoaLumberjack'
      target.build_configurations.each do |config|
        config.build_settings['SWIFT_VERSION'] = 2.3
      end
    end
  end
end

**Comment by reidmain** [comment]
It does not appear that Carthage has the ability to pass a build setting into a dependency. It relies on the Xcode project of every dependency to be setup exactly as they are needed to be successfully compiled using xcodebuild.

Currently I have taken to quickly modifying the Xcode workspace for CocoaLumberjack inside the Carthage/Checkouts folder and then building it.

**Comment by bpoplauschi** [comment]
Should be fixed by #747
